projects/commudle-admin/src/app/app.component.ts

```diff
@@ -1,54 +1,52 @@
-import { CookieConsentService } from './services/cookie-consent.service';
-import { Component, Inject, PLATFORM_ID, OnInit } from '@angular/core';
-import { ApiRoutesService } from 'projects/shared-services/api-routes.service';
-import { environment } from '../environments/environment';
-import { LibAuthwatchService } from 'projects/shared-services/lib-authwatch.service';
-import { NbMenuItem, NbSidebarService, NbWindowService, NbWindowState, NbIconLibraries } from '@nebular/theme';
-import { faBars } from '@fortawesome/free-solid-svg-icons';
-import { ICurrentUser } from 'projects/shared-models/current_user.model';
-import { Router } from '@angular/router';
-import { DOCUMENT, isPlatformBrowser } from '@angular/common';
-import { Title } from '@angular/platform-browser';
-import { ActionCableConnectionSocket } from 'projects/shared-services/action-cable-connection.socket';
-import { UserPersonalDiscussionChatNotificationsChannel } from 'projects/shared-services/websockets/user-personal-discussion-chat-notifications.channel';
-import { CookieConsentComponent } from 'projects/shared-components/cookie-consent/cookie-consent.component';
-import { AppCentralNotificationService } from './services/app-central-notifications.service';
+import {CookieConsentService} from './services/cookie-consent.service';
+import {Component, Inject, OnInit, PLATFORM_ID} from '@angular/core';
+import {ApiRoutesService} from 'projects/shared-services/api-routes.service';
+import {environment} from '../environments/environment';
+import {LibAuthwatchService} from 'projects/shared-services/lib-authwatch.service';
+import {NbIconLibraries, NbMenuItem, NbSidebarService, NbWindowService, NbWindowState} from '@nebular/theme';
+import {faBars} from '@fortawesome/free-solid-svg-icons';
+import {ICurrentUser} from 'projects/shared-models/current_user.model';
+import {Router} from '@angular/router';
+import {DOCUMENT, isPlatformBrowser} from '@angular/common';
+import {Title} from '@angular/platform-browser';
+import {ActionCableConnectionSocket} from 'projects/shared-services/action-cable-connection.socket';
+import {AppCentralNotificationService} from './services/app-central-notifications.service';
+import {CookieConsentComponent} from '../../../shared-components/cookie-consent/cookie-consent.component';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent implements OnInit {
-  private isBrowser: boolean = isPlatformBrowser(this.platformId);
```

Comment:
Why did we move it down?

Reply:
That is due to ESLint styling... Is it better if it's on top? If so, I can change it